plugins/woocommerce/src/Blocks/BlockTypes/ProductGalleryLargeImage.php

```diff
@@ -140,6 +140,7 @@ class="<?php echo esc_attr( $base_classes ); ?>"
 							data-wp-on--touchstart="actions.onTouchStart"
 							data-wp-on--touchmove="actions.onTouchMove"
 							data-wp-on--touchend="actions.onTouchEnd"
+							loading="lazy"
 							alt=""
```

Comment:
I have not used the `decoding` attributes before so I am curious how come it is used in the other instance but not here?

Reply:
Decoding async the large image would increase the LCP ( which is attributed to the large image ) and not the thumbnails where it can be safely used 

https://wordpress.org/support/topic/lcp-problem-images-decoding-async-and-fetchpriorityhigh/